docs/permissions/aws/main.tf

```
@@ -0,0 +1,87 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.3.0"
+    }
```

afaik `terrafomr fmt` is fine with this compression, right?

```suggestion
    aws = { source  = "hashicorp/aws", version = "~> 4.3.0" }
```

---

🤣